documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1659,6 +1685,12 @@ See xref:configuration/avro.adoc#names[Avro naming] for more details.
 |10000 (10 seconds)
 |The number of milli-seconds to wait between retry attempts when the connector fails to connect to a replication slot.
 
+|`toasted.value.placeholder`
+|`__DEBEZIUM_TOASTED_VALUE__`
```

Comment:
@jpechane, one last comment: How about changing the default value into something more neutral, e.g. "__debezium_unavailable_value"? That'd allow for potential re-use later on, e.g. in the context of MySQL's non-full binlog mode. Also for consumers the fact that this is from a TOAST column is mostly a technicality. Note I'd also suggest to lower-case it and only have "__" at the beginning to be consistent with other usages of "__debezium..." literals.